database/migrations/2022_02_02_203008_filesize_size_variants.php

```diff
@@ -14,6 +16,15 @@
  */
 class FilesizeSizeVariants extends Migration
 {
+	private ConsoleOutput $output;
+	private ConsoleSectionOutput $msgSection;
+
+	public function __construct()
+	{
+		$this->output = new ConsoleOutput();
+		$this->msgSection = $this->output->section();
```

Comment:
Would it be possible to use
```suggestion
		$this->msgSection = new ConsoleOutput()->section();
```
and remove the variable in line 19?